src/arrays/qucoeffs.jl

```diff
@@ -0,0 +1,69 @@
+import Base: transpose,
+    ctranspose,
+    conj,
+    size,
+    ndims,
+    length,
+    getindex,
+    setindex!
+
+############
+# QuCoeffs #                 
+############
+    abstract ConjBool{Conj}
+    abstract TranBool{Tran}
```

Comment:
Do we need new types here? Aren't bools sufficient? 

Allowing for conjugation and transposition separately is certainly most general, but do we have a use case for this? I thought that indicating "duality" (whatever that means in the specific case) would be enough (and maybe easier to grok for implementing inner products and such).